mods/ra/maps/negotiations/map.yaml

```diff
@@ -0,0 +1,1351 @@
+MapFormat: 12
+
+RequiresMod: ra
+
+Title: Negotiations
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 128,128
+
+Bounds: 30,40,70,50
+
+Visibility: MissionSelector
+
+Categories: Mission
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: england
+	PlayerReference@USSR:
+		Name: USSR
+		Bot: campaign
+		Faction: soviet
+		Color: FE1100
+		Allies: BadGuy
+		Enemies: Greece, England, GoodGuy
+	PlayerReference@BadGuy:
+		Name: BadGuy
+		Bot: campaign
+		LockFaction: True
+		Faction: soviet
+		Color: FF8800
+		Allies: USSR
+		Enemies: Greece, England
```

Comment:
```suggestion
		Enemies: Greece, England, GoodGuy
```

?

Reply:
(Seems like all the Allies/Enemies setups are a bit off.)

Reply:
That's indeed a mistake, though the two shouldn't come into contact. England and Germany are intentionally a bit skewed, but all the diplomacy could use some changes. Thanks.